### PR TITLE
Upgraded to Laravel 5.3

### PR DESCRIPTION
**Pull request by bkuhl**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,10 @@
 APP_ENV=local
 APP_DEBUG=true
-APP_KEY=SomeRandomString
 APP_URL=http://localhost
 
+# CHANGE THIS KEY and do not use in any deployed environments
+APP_KEY=base64:5dr2i+o8tZzVbtNcjZCtLusNiB6bxgCx2UYLFnXzOQw=
```
Comment (jlaswell) [inline .env.example]: I think we should let CI handle this with the `artisan` command.

Comment (bkuhl) [inline .env.example]: The order of events makes this challenging.  In order to do that we need to have PHP7, which we don't have until we provision the containers, which rely on the .env

**Comment by bkuhl** [comment]
@millidavids @jlaswell 📟

**Comment by jlaswell** [comment]
LGTM!

**Pull request merged**